### PR TITLE
Update plugin language list + update flow's website

### PR DESCRIPTION
**Pull request by jjw24**
Update the plugin language list to include JS and TS

Update url to flowlauncher.com

### CODE REVIEW

**Comment by jjw24** [comment]
@Flow-Launcher/team do we want to add the language logos to make it standout that these languages are available for users to create plugins in?

**Review by onesounds — APPROVED** [review]

**Comment by taooceros** [comment]
the python plugin link is inaccessible.

File: README.md [hunk 1]
```diff
@@ -159,9 +159,15 @@ And you can download <a href="https://github.com/Flow-Launcher/Flow.Launcher/dis
...
 
-- Support wide range of plugins. Visit [here](https://flow-launcher.github.io/docs/#/plugins) for our plugin portfolio.
-- If you are using Python plugins, flow will prompt to either select the location or allow Python (Embeddable) to be automatic downloaded for use.
-- Create and publish your own plugin to flow! Take a look at our plugin development documentation for [C#](https://flow-launcher.github.io/docs/#/develop-dotnet-plugins) or [Python](https://flow-launcher.github.io/docs/#/develop-py-plugins)
+- Support wide range of plugins. Visit [here](https://flowlauncher.com/docs/#/plugins) for our plugin portfolio.
+- Publish your own plugin to flow! Create plugins in:
+
+<p align="center">
+<a href="https://flowlauncher.com/docs/#/develop-dotnet-plugins"><img src="https://user-images.githubusercontent.com/6903107/147870065-4096f233-147c-434e-a3ac-69519582605f.png" width="64"></a>
+<a href="https://flowlauncher.com/docs/#/develop-py-plugins"><img src="https://user-images.githubusercontent.com/6903107/147870066-7599eb15-0333-468e-82e8-4d432ceb5a45.png" width="64"></a>
```
Comment (jjw24) [inline README.md:167]:
```suggestion
<a href="https://www.flowlauncher.com/docs/#/py-develop-plugins"><img src="https://user-images.githubusercontent.com/6903107/147870066-7599eb15-0333-468e-82e8-4d432ceb5a45.png" width="64"></a>
```

**Comment by jjw24** [comment]
good catch, thank you

**Comment by jjw24** [comment]
Could one of you reapprove please @onesounds @taooceros

**Review by onesounds — APPROVED** [review]

**Pull request merged**